test/ui-test/lightspeedRoleGenTest.ts

```diff
@@ -123,8 +123,7 @@ describe("Verify Role generation reset button works as expected", function () {
     await workbenchExecuteCommand("View: Close All Editor Groups");
 
     const notifications = await workbench.getNotifications();
-    for (let i = 0; i < notifications.length; i++) {
-      const n = notifications[i];
+    for (const n of notifications) {
```

Comment:
:+1: and good enough as it is. Note for the future, we have the pattern at multiple places in the tests, it would be nice to have some "reset env" function.